src/pybind/mgr/influx/module.py

```diff
@@ -178,12 +178,12 @@ def get_df_stats(self, now):
         pool_info = {}
 
         df_types = [
-            'bytes_used',
+            'stored',
```

Comment:
@ifed01 you might want to update 

https://github.com/ceph/ceph/blob/590babe82645ca901d69f2a166cb55000890ef8e/qa/tasks/mgr/dashboard/test_health.py#L94

as well.